### ISSUE TITLE
make lint is not working with current chart

### ISSUE THREAD

**Issue by joawin** [jfrog/charts#1714]
**This this a request for help**:

---

**Version of Helm and Kubernetes**:
v3.8.1

**Which chart**:
stable/artifactory-ha

**What happened**:
make lint command is throwing an error
```
==> Linting stable/artifactory-ha
[ERROR] templates/artifactory-service.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 29: could not find expected ':'

Error: 1 chart(s) linted, 1 chart(s) failed

```

Same issue if i just run `make lint -- --charts stable/artifactory
`

**What you expected to happen**:
I expect that make lint is running succesful

**How to reproduce it** (as minimally and precisely as possible):
I have just changed appVersion and version of the Chart
```
git diff
diff --git a/stable/artifactory-ha/Chart.yaml b/stable/artifactory-ha/Chart.yaml
index 17b83e58..6cab7c09 100644
--- a/stable/artifactory-ha/Chart.yaml
+++ b/stable/artifactory-ha/Chart.yaml
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: 7.47.14
+appVersion: 7.47.15
 dependencies:
 - condition: postgresql.enabled
   name: postgresql
@@ -21,4 +21,4 @@ name: artifactory-ha
 sources:
 - https://github.com/jfrog/charts
 type: application
-version: 107.47.14
+version: 107.47.15

```

**Anything else we need to know**:
```
make lint
test/lint-charts.sh
Installing kubeval...
Local run, not downloading kubeval cli...
Installing helm...
Local run, not downloading helm cli...
Error: No such container: ct
Linting charts...

------------------------------------------------------------------------------------------------------------------------
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 artifactory-ha => (version: "107.47.15", path: "stable/artifactory-ha")
------------------------------------------------------------------------------------------------------------------------

"jfrog" has been added to your repositories
"bitnami" has been added to your repositories
"hashicorp" has been added to your repositories
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "hashicorp" chart repository
...Successfully got an update from the "bitnami" chart repository
...Successfully got an update from the "jfrog" chart repository
Update Complete. ⎈Happy Helming!⎈
Saving 1 charts
Downloading postgresql from repo https://charts.jfrog.io/
Deleting outdated charts
Linting chart 'artifactory-ha => (version: "107.47.15", path: "stable/artifactory-ha")'
Checking chart 'artifactory-ha => (version: "107.47.15", path: "stable/artifactory-ha")' for a version bump...
Old chart version: 107.47.14
New chart version: 107.47.15
Chart version ok.
Validating /workdir/stable/artifactory-ha/Chart.yaml...
Validation success! 👍

Linting chart with values file 'stable/artifactory-ha/ci/access-tls-values.yaml'...

==> Linting stable/artifactory-ha
[ERROR] templates/artifactory-service.yaml: unable to parse YAML: error converting YAML to JSON: yaml: line 29: could not find expected ':'

Error: 1 chart(s) linted, 1 chart(s) failed
Error: Error linting charts: Error processing charts
------------------------------------------------------------------------------------------------------------------------
 ✖︎ artifactory-ha => (version: "107.47.15", path: "stable/artifactory-ha") > Error waiting for process: exit status 1
------------------------------------------------------------------------------------------------------------------------
Error linting charts: Error processing charts
make: *** [lint] Error 1
```

**Comment by Logeshwarsn** [comment]
@joawin , We have created a ticket for internal tracking. We will fix this issue shortly.

**Comment by sankalp28** [comment]
@Logeshwarsn any updates on this issue fixing ?

**Comment by Logeshwarsn** [comment]
@joawin Will give you the update by this end of week.

**Comment by Logeshwarsn** [comment]
Hi @joawin,

Its an artifactory-ha chart issue. It will be fixed in upcoming release.

**Comment by chukka** [comment]
Fixed in #1728
